src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final String DEFAULT_COLOR = "WHITE";
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int MAX_NUMBER = 10;
+    private Random random = new Random();
+    private ColorSupplier randomColor = new ColorSupplier();
+    private int randomNum = random.nextInt(MAX_NUMBER);
```

Comment:
this should be local variable

Reply:
I do not quite understand it. Local var are vars declared inside of the methods, but random, randomNum or randomColor being used in different methods isn't it better to declare them as Class var?  Or I understood the reference wrong, sorry for that😊

Reply:
Dont write comments here - because mentors cant see them until your next re-request

Reply:
Random and ColorSupplier - should be class-level
randomNum should be generated each time and you need that new value there - there is no need to have a class level variable. Also that randomNumber has nothing to do with FigureSupplier class